index.html

```diff
@@ -152,8 +162,14 @@
       <script src="./src/simulation.js"></script>
       <script>
 
+        if ('serviceWorker' in navigator) {
+          navigator.serviceWorker.register('./service.js').then(function(serviceWorkerRegistration) {
+            serviceWorkerRegistration.update();
```

Comment:
Here, I force the update of the service worker on every page load but it updates only if new version has different hash. This makes sure that users don't get stuck with old service worker forever.